src/main/resources/config/application.yml

```diff
@@ -20,6 +19,8 @@ server:
     mime-types: application/json,application/xml,text/html,text/xml,text/plain
 
 revocation:
+  authorization:
+    reload: 600000
```

Comment:
Can the time unit be in seconds, as in all other parameters in the revocation service?